src/main/java/org/mifos/connector/ams/pesacore/zeebe/ZeebeWorkers.java

```diff
@@ -65,11 +65,18 @@ public void setupWorkers() {
 
                         producerTemplate.send("direct:transfer-validation-base", ex);
 
-                        boolean isPartyLookUpFailed = ex.getProperty(PARTY_LOOKUP_FAILED, boolean.class);
+                        Boolean isPartyLookUpFailed = ex.getProperty(PARTY_LOOKUP_FAILED, Boolean.class);
                         variables.put(PARTY_LOOKUP_FAILED, isPartyLookUpFailed);
+                        if(isPartyLookUpFailed != null && isPartyLookUpFailed) {
+                            variables.put(ERROR_INFORMATION, ex.getIn().getBody(String.class));
+                            // TODO: add [ERROR_CODE] and [ERROR_DESCRIPTION] variable in zeebe after parsing [ERROR_INFORMATION]
```

Comment:
@avikganguly01 once gilbert will share the payload ill parse the `ERROR_INFORMATION` and update the `ERROR_CODE` and `ERROR_DESCRIPTION` variable.

Reply:
I guess Gilbert mentioned it looks like this.

 { "400": {
                 "description": "Bad Request, incorrect data or missing required data"
             },
             "401": {
                 "description": "check API authentication"
             },
             "404": {
                 "description": "Client Not Found"
             }
}